strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/storage/validation/version/NugetReleaseVersionValidatorTest.java

```diff
@@ -0,0 +1,62 @@
+package org.carlspring.strongbox.storage.validation.version;
+
+import org.carlspring.strongbox.artifact.coordinates.ArtifactCoordinates;
+import org.carlspring.strongbox.artifact.coordinates.NugetHierarchicalArtifactCoordinates;
+import org.carlspring.strongbox.storage.repository.Repository;
+import org.carlspring.strongbox.storage.repository.RepositoryPolicyEnum;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.fail;
+
+public class NugetReleaseVersionValidatorTest
+{
+
+    Repository repository = new Repository();
+
+    NugetReleaseVersionValidator validator = new NugetReleaseVersionValidator();
+
+    @Before
+    public void setUp()
+            throws Exception
+    {
+        repository.setPolicy(RepositoryPolicyEnum.RELEASE.toString());
+    }
+
+    @Test
+    public void testReleaseValidation()
+            throws VersionValidationException
+    {
+        ArtifactCoordinates coordinates1 = new NugetHierarchicalArtifactCoordinates();
+        coordinates1.setVersion("1");
+
+        ArtifactCoordinates coordinates2 = new NugetHierarchicalArtifactCoordinates();
+        coordinates2.setVersion("1.0");
+
+        ArtifactCoordinates coordinates3 = new NugetHierarchicalArtifactCoordinates();
+        coordinates3.setVersion("1.0-RELEASE");
```

Comment:
We should not promote the use of versions like `1.0-RELEASE`. If it's a release, it will be `1.0`, no need for extra noise in the version. Perhaps you were looking for another good example, so here are some:
* `1.0-rc-1`
* `1.0-milestone-1`
* `1.0-alpha-1`
* `1.0-beta-1`